### PR TITLE
Issue 332 - do not cleanup source if DST is down

### PR DESCRIPTION
**Pull request by JMoVS**
This is the pull request for issue #322 (I've been bitten by the same behaviour just now) and I checked the pull requests and @healthian pushed it to its own fork but never made a pull reuqest.

@healthian if you need help with the PR, please say so and I can guide you through. This PR uses your commit and your author and commiter info, it's just me requesting from my repo so this can actually get merged.

### CODE REVIEW

**Comment by healthian** [comment]
Thanks for getting that; I still have not worked my way back to this task yet. I needed to look up how to do a pull request on github, and test my change. When I was looking into it, I was rushing, and checked it into my fork with a missing semicolon on this line:

`next if ($sendFailed)`

**Comment by JMoVS** [comment]
@healthian Hi, I just pushed (amended to your commit the semikolon and force-pushed to the branch) an updated version. Essentially, if you click the "pull request" button on Github, you'Re good to go. If you're fine with me doing the pull request (commit still lists you as author), then that's fine with me too. ;-)

**Comment by JMoVS** [comment]
Now we only need @oetiker to have a look ;-)

**Comment by healthian** [comment]
Yes, and thanks for taking care of it!

**Comment by healthian** [comment]
I fixed my typo, but I will need a couple hours to test it out. I don't want to submit a pull request until I know it works. It is a trivial change, but even a trivial typo can doom a fix, and hence, testing is always needed. LOL.

File: lib/ZnapZend.pm [hunk 1]
```diff
@@ -273,6 +274,7 @@ my $sendRecvCleanup = sub {
                 $backupSet->{"dst_$key" . '_valid'} or
                     $self->zLog->warn("destination '" . $backupSet->{"dst_$key"}
                         . "' does not exist or is offline. ignoring it for this round...");
+                $sendFailed = 1;
```
Comment (oetiker) [inline lib/ZnapZend.pm]:
```perl
... or do {
    $self->zLog->warn("destination '" . $backupSet->{"dst_$key"}
                         . "' does not exist or is offline. ignoring it for this round...");
    $sendFailed = 1;
};

**Comment by JMoVS** [comment]
Did I change it accordingly or are there further changes needed?

**Comment by oetiker** [comment]
thank you for the patch

**Pull request merged**